apps/counter/main.py

```diff
@@ -0,0 +1,99 @@
+# coding=utf-8
+from time import time
+
+from input.input import InputListener
```

Comment:
Does your IDE need this imported for autocomplete? Since this import is not actually used anywhere

Reply:
Yes, it can be removed though. What do you think ?